### DISCUSSION TITLE
Fist Time User - Joystick Keysets

### DISCUSSION THREAD

**Discussion by DSC-david** [dirkwhoffmann/virtualc64#708] [Q&A]
Good Evening!

I am a first time user of VirtualC64 and it is a great piece of software.

However I am not able to set the Emulation Keys for the Keysets for joysticks in the interface.

Is there something I am missing here?

In addition, is there any documentation or a User's Manual or something in that vein?

Thanks in advance,
D

**Comment by neleabels** [comment]
Just in case somebody else is having the question (it took me some minutes to find the answer myself), you can define the controls under "preferences/controls". Just klick the buttons on the right side (e.g. for "keyset 1 - joystick up") and then push the key which you would like to use. In this case it's "arrow up" with the code 7E.
![Bildschirm­foto 2023-01-28 um 08 45 31](https://user-images.githubusercontent.com/123799276/215254127-2befbd56-42bc-4b57-9cef-c77c439bfc7e.png)
